### PR TITLE
[dav1d] add dav1d project

### PR DESCRIPTION
**Pull request by jannau**
dav1d is an AV1 decoder

http://www.jbkempf.com/blog/post/2018/Introducing-dav1d

Mozilla is planning to use the decoder in Firefox
https://bugzilla.mozilla.org/show_bug.cgi?id=1493397

### CODE REVIEW

**Comment by jannau** [comment]
On 2018-10-05 09:18:49 -0700, googlebot wrote:
> 
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> :memo: **Please visit <https://cla.developers.google.com/> to sign.**
> 
> Once you've signed (or fixed any issues), please reply here (e.g. `I signed it!`) and we'll verify it.

I signed it

best regards

File: projects/dav1d/project.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+homepage: "https://code.videolan.org/videolan/dav1d"
+primary_contact: "janne.grunau@gmail.com"
+auto_cc: "rsbultje@gmail.com"
```
[the remark this answers is not in the document]
Comment (jannau) [reply projects/dav1d/project.yaml]: indeed, I pushed a fixed version

File: projects/dav1d/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER janne-vlc@jannau.net
+RUN apt-get update && apt-get install -y curl python3-pip && \
+    pip3 install meson ninja
+RUN curl --silent -O https://storage.googleapis.com/aom-test-data/fuzzer/dec_fuzzer_seed_corpus.zip
+RUN curl --silent -O http://jannau.net/dav1d_fuzzer_seed_corpus.zip
+RUN git clone --depth 1 https://code.videolan.org/videolan/dav1d.git dav1d
```
[the remark this answers is not in the document]
Comment (jannau) [reply projects/dav1d/Dockerfile]: our fuzz target ignores the whole ivf header (the first 35 bytes) and libaom dictionary has only two strings from the header.

File: projects/dav1d/project.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+auto_cc: "dav1d-fuzz@videolan.org"
+sanitizers:
+  - address
+  - memory
+experimental: True
```
[the remark this answers is not in the document]
Comment (jannau) [reply projects/dav1d/project.yaml]: We would like to keep the experimental flag until we start preparing the first release. dav1d is still in it's early development phase and timed disclosure process looks at this point as unnecessary overhead. We will of course credit the found bugs to oss-fuzz.

[the remark this answers is not in the document]
Comment (jonathanmetzman) [reply projects/dav1d/project.yaml]: OK, but IMO it is much easier to get notified when there are bugs.

**Pull request merged**